core/input/mouse.js

```diff
@@ -108,27 +116,74 @@ Mouse.prototype = {
         this._handleMouseButton(e, 0);
     },
 
+    // Mouse wheel events are sent in steps over VNC. This means that the VNC
+    // protocol can't handle a wheel event with specific distance or speed.
+    // Therefor, if we get a lot of small mouse wheel events we combine them.
+    _generateWheelStepX: function (pos) {
+
+        if (this._accumulatedWheelDeltaX < 0) {
```

Comment:
Any scenario where we might need to send multiple events?

Reply:
Absolutely, but I think that is another discussion. That could make the behavior deviate from how things worked previously, even when using an input device that does not send many small delta events.

A goal for this was to keep the old scroll wheel behavior where it wasn't too sensitive.